lib/kennel/unmuted_alerts.rb

```diff
@@ -44,7 +48,7 @@ def time_since(t)
       end
 
       def filtered_monitors(api, tag)
-        # Download all monitors with given tag
+        # Download all monitors with given tag and scope
```

Comment:
has nothing to do with scope
```suggestion
        # Download all monitors with given tag
```